data/tweaks/mtu-probing/metadata.yml

```diff
@@ -0,0 +1,3 @@
+description: "Enable MTU probing (fixes some applications like Ubisoft Connect)"
```

Comment:
you may want separate titles and descriptions, as per my last comment about putting info in warnings